### DISCUSSION TITLE
how to change default selected stores and change default selected series

### DISCUSSION THREAD

**Discussion by 100to1000aday** [jef/streetmerchant#1077] [Help]
### Description

how to choose selected stores change default and add change selected series its on default what file do I edit to change default

**Comment by Niswendel** [comment]
open the env file using notepad++, then edit the stores

**Comment by 100to1000aday** [comment]
Yeah but it’s saying nvidia is default selected store how do you make a store default store and make default series

**Comment by Splode** [comment]
@100to1000aday For example, copy the `.env-example` as `.env` and edit the following lines:

```
# .env

SHOW_ONLY_SERIES="3080"
STORES="adorama,amazon,asus,bandh,bestbuy,corsair,gamestop,officedepot,pny,target,walmart,zotac"
```
